source/js/components/inputItemGrid.js

```diff
@@ -1,25 +1,37 @@
 import React from 'react';
+import {connect} from 'react-redux';
 import InputItem from './inputItem.js';
 import {TransitionGroup} from 'react-transition-group';
 import FadeAndSlideTransition from './fadeAndSlideTransition.js';
 
-function List({children}) {
-  return (
-    <ul className="input-item-grid">
-      {children}
-    </ul>
-  )
-}
+const mapStateToProps = state => {
+	return { inputItems: state.inputItems };
+};
 
-const InputItemGrid = ({items, onRemoveInput}) => (
+const List = ({children}) => {
+	return (
+		<ul className="input-item-grid">
+			{children}
+		</ul>
+	);
+};
+
+const InputItemGrid = ({inputItems}) => (
 	<main className="main">
 		<div className="wrapper">
 			<TransitionGroup component={List}>
 				{
-					items.map(item => {
+					inputItems.map(item => {
 						return (
 							<FadeAndSlideTransition duration={250} key={item.id}>
-								<InputItem key={item.id} id={item.id} value={item.name} inputType={item.inputType} onRemoveInput={onRemoveInput} />
+								<InputItem 
+									key={item.id} 
+									id={item.id} 
+									value={item.name} 
```

Comment:
Looks like the "value" property isn't used in the InputItem component. Can it be removed? If so, you could use the spread operator on the item variable to populate the props on InputItem (but you'd still need to leave the key prop in place).